### PR TITLE
@l2succes => Bring updateConversation up to Relay spec

### PR DESCRIPTION
**Pull request by mzikherman**
screen shot 2017-04-06 at 5 56 35 pm

### CODE REVIEW

**Comment by mzikherman** [comment]
This does https://github.com/artsy/metaphysics/pull/598 but for `updateConversation`. Basically it moves all the inputs into a single `input` arg (of type `UpdateConversationInput`), and creates the required `UpdateConversationPayload` for Relay.

File: schema/me/conversations.js [hunk 1]
```diff
@@ -39,52 +39,54 @@ export const BuyerOutcomeTypes = new GraphQLEnumType({
...
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  to_type: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  to_name: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  buyer_outcome: {
+    type: GraphQLString,
```
Comment (l2succes) [inline schema/me/conversations.js]: Should we go ahead and change the type to `BuyerOutcomeType` or is that something different?

Comment (mzikherman) [inline schema/me/conversations.js]: We could, yea.

Comment (l2succes) [inline schema/me/conversations.js]: Ok, i'll merge it as is for now. We can switch the type later.

**Pull request merged**